### PR TITLE
Adds tests for invalid client and server combinations

### PR DESCRIPTION
**Pull request by bretterer**

### CODE REVIEW

File: packages/@okta/vuepress-theme-default/layouts/Quickstart.vue [hunk 1]
```diff
@@ -100,10 +100,15 @@
       this.activeClient = matches && matches[1];
       this.activeServer = matches && matches[2];
       this.activeFramework = matches && matches[3];
+
```
Comment (swiftone) [inline packages/@okta/vuepress-theme-default/layouts/Quickstart.vue]: nit: Why so much space before a closing curly?

File: tests/selenium/spec/quickstarts-spec.js [hunk 2]
```diff
@@ -47,6 +47,149 @@ describe('quickstarts page default selections spec', () => {
...
+    expect(await quickstartsPage.getActiveLinkText())
+      .to.include.members([
+      'Okta Sign-In Page',
+      'Node JS',
+      'Express.js'
+    ]);
+  }));
+
+  it('selects okta-sign-in-page + nodejs + express if invalid server framework in hash', util.itHelper(async () => {
+    await quickstartsPage.navigate('/quickstart/#/okta-sign-in-page/nodejs/null');
```
Comment (swiftone) [inline tests/selenium/spec/quickstarts-spec.js]: `/null`?   Really?  Weird.

Comment (bretterer) [reply tests/selenium/spec/quickstarts-spec.js]: Yup... this is what was coming up in the url which caused this test!  fun times :)

File: packages/@okta/vuepress-theme-default/layouts/Quickstart.vue (hunk 1 above)
Comment (bretterer) [reply packages/@okta/vuepress-theme-default/layouts/Quickstart.vue]: Removed Code, missed line feeds

**Pull request merged**